deepchem/utils/batch_utils.py

```diff
@@ -129,3 +130,159 @@ def batch_coulomb_matrix_features(X_b: np.ndarray,
     dist_mem_j = np.concatenate(distance_membership_j).astype(np.int64)
     features = [atom_number, gaussian_dist, atom_mem, dist_mem_i, dist_mem_j]
     return features
+
+
+def batch_elements(elements: Any, batch_size: int):
+    """Combine elements into batches.
+
+    Parameters
+    ----------
+    elements: Any
+        Elements to be combined into batches.
+    batch_size: int
+        Batch size in which to divide.
+
+    Returns
+    -------
+    batch: Any
+        Batch of elements.
+
+    Examples
+    --------
+    >>> import deepchem as dc
+    >>> # Prepare Data
+    >>> inputs = [[i, i**2, i**3] for i in range(10)]
+    >>> # Run
+    >>> output = list(dc.utils.batch_utils.batch_elements(inputs, 3))
+    >>> len(output)
+    4
+
+    """
+    batch = []
+    for s in elements:
+        batch.append(s)
+        if len(batch) == batch_size:
+            yield batch
+            batch = []
+    if len(batch) > 0:
+        yield batch
+
+
+def create_input_array(sequences: Collection, reverse_input: bool,
+                       batch_size: int, input_dict: Dict, end_mark: Any):
+    """Create the array describing the input sequences for a batch.
+
+    It creates a 2d Matrix empty matrix according to batch size and max_length.
```

Comment:
Why do we want to do this? Can you provide more context on the usage in seq2seq or other models?

Reply:
Done